### ISSUE TITLE
Welcome message page for the first logged in users

### ISSUE THREAD

**Issue by livike** [CodeCatz/TrackCat#234]
After the first login would be nice to redirect to some welcome message, to promt them to fill in their profile or a message that a staff member will review their membership...

**Comment by goranche** [comment]
yup, would be nice... maybe next week, don't introduce new features now :wink:

**Comment by livike** [comment]
:+1:
